README.md

```diff
@@ -618,7 +618,7 @@ Arguments:
 - `dialogTitle`: a title for the save dialog.
 
 Note the inversion of control here:
-JS Paint calls your `systemHooks.showSaveFileDialog` function, and then you calls JS Paint's `getBlob` function.
+JS Paint calls your `systemHooks.showSaveFileDialog` function, and then calls JS Paint's `getBlob` function.
```

Comment:
"you calls" is grammatically incorrect, but the "you" is important here. You, as a user of the API, are meant to call `getBlob`, whereas you _define_ the `systemHooks.showSaveFileDialog` function. It could say "you call" or "your code calls".

Reply:
Addressed in https://github.com/1j01/jspaint/pull/330/commits/7e7b9c0c94fec4003aaff5372d1f8eccf91c07af
Thanks for the great app!